hardware:
  protocol: modbus
  slave_id: 127
  log_level: info
  ensure_physical_mode: true
  ctrl_param_duration_ms: 10
  position_command_scale: 572.9577951308232
  position_state_scale: 0.0017453292519943296
  velocity_state_scale: 0.0017453292519943296
  position_device_min: 0.0
  position_device_max: 1000.0
  port: /dev/ttyUSB1
  baudrate: 460800
  auto_detect: false
  auto_detect_quick: true
  auto_detect_port: ""

